pkg/provider/onepasswordsdk/client.go

```diff
@@ -380,14 +380,12 @@ func (p *Provider) SecretExists(_ context.Context, _ esv1.PushSecretRemoteRef) (
 	return false, fmt.Errorf("not implemented")
 }
 
-// Validate checks if the client is configured correctly
-// currently only checks if it is possible to list vaults.
+// Checks that the client was initialized correctly.
+// We don't make calls the the API, as the rate-limit of the 1password SDK is quite restrictive.
```

Comment:
this looks to me like it should just be storing the result or track a client side limit